docs/sql-guide/statements/statement-insert-bulk-csv-example.md

```diff
@@ -0,0 +1,68 @@
+---
+title: BULK INSERT CSV example
+layout: default
+parent: Statements
+grand_parent: SQL guide
+---
+
+# Ingest a CSV with BULK INSERT
+
+
+## Before you begin
+
+{% include /sql-guide/bulk-insert-eg-before-begin.md%}
+
+## Step 1: create table
+
+```sql
+CREATE TABLE age (
+    _id STRING,
+    name STRING,
+    description STRING,
+    gender STRING,
+    country STRING,
+    occupation STRING,
+    birth_year INT min -32767 max 32767,
+    death_year INT min -32767 max 32767,
+    death_manner STRING,
+    birth_age INT min -32767 max 32767
+) keypartitions 12 shardwidth 65536;
```

Comment:
This looks old. Can you pull this from whats in the current repo?

Reply:
updated as advised

Reply:
Well they do say you learn by doing and failing. So I've nailed this one by now I hope :D